src/local_sensitivity/sensitivity_interface.jl

```diff
@@ -22,6 +22,19 @@ function _adjoint_sensitivities(sol,sensealg,alg,g,t=nothing,dg=nothing;
     adj_sol[end][(1:l) .+ length(sol.prob.u0)]'
 end
 
+function _adjoint_sensitivities(sol,sensealg::SteadyStateAdjoint,alg,g,dg=nothing;
+                                   abstol=1e-6,reltol=1e-3,
+                                   kwargs...)
+  cache = SteadyStateAdjointSensitivityFunction(g,sensealg,sol,dg)
+
+  linsolve_tmp = cache.J'\vec(cache.gu')

```

Comment:
all that should we need to do is compute the vector'*Jacobian, which is the vecjacobian! stuff of the other adjoints. Copy that general workflow, since we want to use backprop on f for that. You might want to read some of the 18.337 notes, specifically https://mitmath.github.io/18337/lecture10/estimation_identification on vjps

Reply:
Thanks for the remarks and the link to the notes! I'll read them and try to update the files accordingly.